site/en/blog/third-party-scripts/index.md

```diff
@@ -0,0 +1,264 @@
+
+-   Third-parties are a cross-cutting problem involving ads, analytics, tag management, utilities, and many others. Each area requires consideration of a unique set of requirements and trade-offs. For example,
+-   The decision to optimize the loading of ads depends on a trade-off between revenue and user experience. Too early—they block valuable content; too late—the user would miss seeing them.
+-   Analytics scripts add to the page weight but provide valuable data about user actions to the business.
+
+We hope to partner with various categories of third-parties, grasp the nuances involved, resolve trade-offs, and develop incentives that work for all. We realize that we have to work separately with entities in every area for our strategy to be effective. This includes our internal partners such as Google Tag Manager, Google Ads, and YouTube.
+
+2.  We want to provide deeper attribution to both site developers and third-party developers. This requires a conscientious effort where we identify what data is most relevant in measuring impact, attribute it accurately and granularly, and provide the right path forward. Ultimately, the calculation of how a given third-party performs against its competition should be transparent to all.
+
+3.  We propose to enhance Chromium so that it can apply optimizations that the user has opted in for. A valuable change becomes available as a standard across all browsers, but it takes time. For example, the [loading](https://developer.mozilla.org/docs/Web/HTML/Element/img#attr-loading) attribute for images and iframes has been available in Chrome/Edge since 2019, but became available in Safari only in 2022. Until a feature is standardized, third-party users will have to ensure that they have optimized for other browsers too. We will help by highlighting this in our developer tools.
+
+4.  To execute on this project, we will have to recruit partners and developers to not only help us understand specific requirements, but also to test experimental solutions at scale, provide feedback, iterate and improvise as and when required. The changes will have to be planned, allowing for a reasonable time frame for testing and evaluation.
+
+## Initial Standards Proposals
+
+We have performed some initial experiments to develop features that can be enabled to optimize the third-party loading process. We are pleased with the results observed and can presently discuss two of these features.
+
+1.  **LazyEmbeds**
+
+Chrome would previously [lazy-load](https://blog.chromium.org/2019/10/automatically-lazy-loading-offscreen.html) offscreen images and iframes for our [lite-mode](https://support.google.com/chrome/answer/2392284?hl=en&co=GENIE.Platform%3DAndroid) users. This feature could be extended to all users to defer the loading of iframes determined to be third-party embeds until the user scrolls near them. This could speed up the loading of other parts of the page, improve Core Web Vitals, reduce memory usage and save data.
```

Comment:
```suggestion
Chrome would previously [lazy-load](https://blog.chromium.org/2019/10/automatically-lazy-loading-offscreen.html) offscreen `<img>` and `<iframe>` elements for our [lite-mode](https://support.google.com/chrome/answer/2392284?hl=en&co=GENIE.Platform%3DAndroid) users. This feature could be extended to all users to defer the loading of `<iframe>` elements determined to be third-party embeds until the user scrolls near them. This could speed up the loading of other parts of the page, improve Core Web Vitals, reduce memory usage and save data.
```